tests/client/core/i18n/test_utils.js

```diff
@@ -445,9 +445,34 @@ describe('i18n utils', () => {
       expect(i18n.formatNumber(9518231)).toEqual('9,518,231');
     });
 
-    it('localised formatting a number', () => {
+    it('provides a number localized by locale', () => {
+      const numberFormatSpy = sinon.spy(Intl, 'NumberFormat');
       const i18n = utils.makeI18n({}, 'de', FakeJed);
       expect(i18n.formatNumber(9518231)).toEqual('9.518.231');
+      sinon.assert.calledWith(numberFormatSpy, 'de');
+    });
+
+    it('calls Intl.NumberFormat.format', () => {
+      const formatStub = sinon.stub();
+      const fakeIntl = {
+        NumberFormat: () => {
+          return {
+            format: formatStub,
+          };
+        },
+      };
+      const i18n = utils.makeI18n({}, 'de', FakeJed, { _Intl: fakeIntl });
+      i18n.formatNumber(9518231);
+      sinon.assert.calledWith(formatStub, 9518231);
```

Comment:
I don't really see how this test is any different from the one above where you just spy on the polyfilled global like `sinon.spy(Intl, 'NumberFormat')`. You could delete this test IMO.